compass/src/components/Shared/StatusBadge/StatusBadge.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Badge } from 'fundamental-react/lib/Badge';
+
+class StatusBadge extends React.Component {
+  PropTypes = {
+    status: PropTypes.string.isRequired,
+  };
+
+  processStatus() {
+    const { status } = this.props;
+    let type = 'warning';
```

Comment:
```suggestion
    let type;
```
This will do alone, as it's gonna be overridden below.